### ISSUE TITLE
TravisCI builds failing with latest Miniconda

### ISSUE THREAD

**Issue by tbekolay** [conda/conda#6487]
We've been successfully using `http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh` in our TravisCI builds for a few years now. Today we ran into issues on all of our builds that use Miniconda; see [this collection of builds](https://travis-ci.org/nengo/nengo/builds/318219757).

What seems to be happening is that the `conda install` command will exit out after a short time (the time is so short that it definitely is not something to do with TravisCI timing out). Here are a few of the failures:

```
libgfortran-1. 100% |###############################| Time: 0:00:00  25.80 MB/s
expat-2.2.5-he 100% |###############################| Time: 0:00:00  35.16 MB/s
gmp-6.1.2-h6c8 100% |###############################| Time: 0:00:00  40.12 MB/s
icu-58.2-h9c2b  86% |##########################     | Time: 0:00:00  52.icu-58.2-h9c2b 100% |###############################| Time: 0:00:00  57.68 MB/s
icu-58.2-h9c2b   7% |##        icu-58.2-h9c2b 100% |###############################| Time: 0:00:00 362.84 MB/s
icu-58.2-h9c2b   7% |##        

The command "if [[ -n "$CONDA_DEPS" ]]; then eval conda install "$CONDA_DEPS"; fi" failed and exited with 1 during .
```

```
intel-openmp-2 100% |###############################| Time: 0:00:00  32.65 MB/s
libgfortran-ng 100% |###############################| Time: 0:00:00  36.97 MB/s
expat-2.2.5-he 100% |###############################| Time: 0:00:00  33.70 MB/s
gmp-6.1.2-h6c8 100% |###############################| Time: 0:00:00  38.05 MB/s
icu-58.2-h9c2b 100% |###############################| Time: 0:00:00  38.94 MB/s
jpeg-9b-h024ee 100% |###############################| Time: 0:00:00  48.02 MB/s
libsodium-1.0. 100% |###############################| Time: 0:00:00  37.78 MB/s
libxcb-1.12-hc 100% |###############################| Time: 0:00:00  37.72 MB/s
mkl-2018.0.1-h   5% |mkl-2018.0.1-h 100% |###############################| Time: 0:00:00 595.82 MB/s
mkl-2018.0.1-h   1% |                               | Time: 0:0mkl-2018.0.1-h 100% |###############################| Time: 0:00:00   1.70 GB/s
mkl-2018.0.1-h   1% |                               | Time: 0:0mkl-2018.0.1-h 100% |###############################| Time: 0:00:00   1.73 GB/s

The command "if [[ -n "$CONDA_DEPS" ]]; then eval conda install "$CONDA_DEPS"; fi" failed and exited with 1 during .
```

This last one is the only one that actually raises an error rather than failing silently.

```
intel-openmp-2 100% |###############################| Time: 0:00:00  17.46 MB/s
expat-2.2.5-he 100% |###############################| Time: 0:00:00  51.36 MB/s
gmp-6.1.2-h6c8 100% |###############################| Time: 0:00:00  50.32 MB/s
icu-58.2-h9c2b  85% |##########################     | Time: 0:00icu-58.2-h9c2b 100% |###############################| Time: 0:00:00  61.66 MB/s
icu-58.2-h9c2b   7% |##        icu-58.2-h9c2b 100% |###############################| Time: 0:00:00 394.33 MB/s
icu-58.2-h9c2b   7% |##        
CondaError: IOError(11, 'Resource temporarily unavailable')
CondaError: IOError(11, 'Resource temporarily unavailable')
CondaError: IOError(11, 'Resource temporarily unavailable')

icu-58.2-h9c2b 100% |###############################| Time: 0:00:00 358.74 MB/s

The command "if [[ -n "$CONDA_DEPS" ]]; then eval conda install "$CONDA_DEPS"; fi" failed and exited with 1 during .
```

Any ideas as to what might be going on? The full build logs can be seen [here](https://travis-ci.org/nengo/nengo/builds/318219757).

**Comment by kalefranz** [comment]
This is a duplicate of #6473.  I really don't know what's going on here.  We haven't changed miniconda in some time.  We did just now release conda 4.3.31, but 4.3.30 has been out for months.

I believe it's a deeper issue than conda has control over.  But I looked at TravisCI build worker and host information for a different project,

* here: https://travis-ci.org/simpeg/discretize/jobs/316579546
* and here: https://travis-ci.org/simpeg/discretize/jobs/316255169

And I didn't see anything different in TravisCI's configuration either.

I really don't know where this is coming from all the sudden.

**Comment by tbekolay** [comment]
According to https://github.com/travis-ci/travis-ci/issues/4704 this has been an issue for some time, and happens when writing to stdout gives an `EAGAIN` error. [This comment](https://github.com/travis-ci/travis-ci/issues/4704#issuecomment-348435959) suggests turning off `O_NONBLOCK` using `fcntl`.

From what I've observed, the issue is somehow related to the amount of writing done to stdout either per-process or per-line... As such, we figured that progress bars output during `conda install` are what's causing the failures. We've added the `--quiet` flag to our builds and they're now working, so I'm going to close this issue.
